script/releases/release-template/2-enqueueTimelock.s.sol

```diff
@@ -51,7 +54,8 @@ contract Queue is MultisigBuilder {
         return _opsCalls;
     }
 
-    function zeusTest() public override {
+    function testDeploy() public virtual {
```

Comment:
I believe we use the `test_Deploy()` style for test cases. i.e `test_*`

Reply:
linter reports that as wrong for me...
<img width="709" alt="image" src="https://github.com/user-attachments/assets/e17aa67c-1bd4-44f0-a39d-0a48d650d4c6">